CHANGELOG.md

```diff
@@ -591,6 +591,7 @@ Added
 - `Control.Validation`, a library for dependent types input validation.
 - `System.Console.GetOpt`, a library for specifying and parsing command line
   options.
+- `Data.List.Sufficient`, a small library defining a structurally inductive view of lists
```

Comment:
This is the CHANGELOG section for version 0.4.0

Reply:
Thanks, I tried finding the section way without paying enough attention.